exp/ipam/internal/webhooks/ipaddress.go

```diff
@@ -123,14 +123,15 @@ func (webhook *IPAddress) validate(ctx context.Context, ip *ipamv1.IPAddress) er
 			))
 	}
 
-	_, err = netip.ParseAddr(ip.Spec.Gateway)
-	if err != nil {
-		allErrs = append(allErrs,
-			field.Invalid(
-				specPath.Child("gateway"),
-				ip.Spec.Gateway,
-				"not a valid IP address",
-			))
+	if ip.Spec.Gateway != "" {
```

Comment:
While our webhook now tolerates an empty gateway it's still a required field in the OpenAPI schema.
I think we need the following there:
```go
	// Gateway is the network gateway of the network the address is from.
	// +optional
	Gateway string `json:"gateway,omitempty"`
```